src/glfw-sandbox/drawings.h

```diff
@@ -0,0 +1,88 @@
+#pragma once
+#include <windows.h>
+#include <GL/gl.h>
+#include <GL/glu.h>
+#include <math.h>
+
+
+class drawObjects
+{
+private:
+	/*void triangle();
+	void wheel(int lod);
+	void grid(float xSize, float zSize);*/
+
+public:
+
+	void triangle()
```

Comment:
Tip: Do the drawing methods require any instance data? If not, how can we decorate them?

Reply:
When you think about it, no, they don't need to be instanced.

But maybe calling them via classes without instancing first?
Not sure what do you mean by decorating them :(

Reply:
In Python, when method does not use instance data, you decorate it with @staticmethod.

I would just suggest to do C++ analogy here :)

Reply:
Well... templates comes to mind but not sure if they are the right fit there.

For calling functions without instancing we could use DrawObjects::triangle()

Reply:
`DrawObjects::triangle();` is perfectly fine :)